libplatsupport/src/plat/bcm2711/system_timer.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2019, Data61, CSIRO (ABN 41 687 119 230)
+ * Copyright (C) 2021, Hensoldt Cyber GmbH
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+#include <errno.h>
+#include <stdlib.h>
+#include <utils/util.h>
+#include "../../arch/arm/clock.h"
+#include <platsupport/timer.h>
+#include <platsupport/plat/system_timer.h>
+
+/*
+ * The system timer on the BCM283[5-7] is fairly simple in its nature.
```

Comment:
We are on bcm2711 here, so better state explicitly that we have copied the settings from there. Otherwise it seemly a bit that this was a blind copy&paste thing that nobody really looked into

Reply:
Added the following comment:
https://github.com/seL4/util_libs/pull/60/files#diff-670b03b687221d356abaeb955f3b6cc793aa8a991e467d697ac76b82e5eccd73R15-R17

Reply:
Look ok. I have create https://github.com/seL4/util_libs/issues/61 as a follow-up action once all this is merged to look into reducing the code duplication